mitodl/ocw-hugo-themes
# refactor: old course theme removed

#### Pre-Flight checklist


#### What are the relevant tickets?
closes https://github.com/mitodl/ocw-hugo-themes/issues/986

#### What's this PR do?
- Removes old course theme

#### How should this be manually tested?
- Checkout this branch
- Build any course locally
- Verify that the course builds successfully
- Now test the course thoroughly and verify that everything is on point
- Repeat this for multiple courses
